spring-boot-modules/spring-boot-crud/src/main/java/com/baeldung/entitydtodifferences/dto/BookDto.java

```diff
@@ -0,0 +1,30 @@
+package com.baeldung.entitydtodifferences.dto;
+
+import com.fasterxml.jackson.annotation.JsonProperty;
+
+public class BookDto {
+
+    @JsonProperty("NAME")
+    private final String name;
+
+    @JsonProperty("AUTHOR")
+    private final String author;
+
+    // Default constructor for Jackson deserialization
+    public BookDto() {
+        this(null, null);
```

Comment:
Let's omit this line. It is the default behavior of Java objects to initialize their Object fields to null.